ports/rp2/modules/_boot_fat.py

```diff
@@ -6,9 +6,10 @@
 bdev = rp2.Flash()
 try:
     fs = vfs.VfsFat(bdev)
+    vfs.mount(fs, "/")
```

Comment:
Maybe this (and the corresponding lines in the "except" below) can be `vfs.mount(vfs.VfsFat(bdev), "/")`, and then the `fs` variable is not needed and the `del fs` can be removed below.

Reply:
Done.